cachito/workers/__init__.py

```diff
@@ -29,22 +32,22 @@ def run_cmd(cmd, params, exc_msg=None):
     params.setdefault("capture_output", True)
     params.setdefault("universal_newlines", True)
     params.setdefault("encoding", "utf-8")
+    with tracer.start_as_current_span("running cmd " + " ".join(cmd)):
```

Comment:
The reason we're doing this inline instead of using the decorator is so that we can also track the actual `cmd`? I think it'd be better to avoid doing inline operations as much as we can, but if there's no way to capture this info, I'm fine with this.